### PR TITLE
Extension installation for FIE ampdocs

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: src/service.js [hunk 1]
```diff
@@ -114,6 +118,16 @@ export function installServiceInEmbedScope(embedWin, id, service) {
   );
   registerServiceInternal(embedWin, embedWin, id, () => service);
   getServiceInternal(embedWin, id); // Force service to build.
+  const ampdocFieExperimentOn = isExperimentOn(topWin, 'ampdoc-fie');
```
Comment (choumx) [inline src/service.js]: Should we put the above code in an `else` block?

Comment (dvoytenko) [reply src/service.js]: I figured this way it'd be easier to cleanup the experiment since we won't need to remove the `else` block later.

Comment (choumx) [reply src/service.js]: Might be good to avoid unintended side effects from registering the service on the window though.

Comment (dvoytenko) [reply src/service.js]: Oh... I see. Misunderstood. Will send a separate pull request for this specifically.

**Pull request merged**